src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PremisesTest.kt

```diff
@@ -1179,23 +1168,21 @@ class PremisesTest : IntegrationTestBase() {
           probationRegionEntityFactory.produceAndPersist { withYieldedApArea { apAreaEntityFactory.produceAndPersist() } }
         }
         withService("CAS1")
-        withTotalBeds(20)
-      }
+      }.onEach { addRoomsAndBeds(it, 5, 4) }
```

Comment:
Nice little helper here 👍

Reply:
Might be good to have the room and bed counts as named parameters so they're not so "magic number"-y

Reply:
Good idea, i've done this 👍